src/components/cart/CartTotalBox/CartTotalBox.jsx

```diff
@@ -2,32 +2,32 @@ import React from "react";
 
 import "./CartTotalBox.css";
 
-const CartTotalBox = () => {
+const CartTotalBox = ({item}) => {
   return (
     <div class="total-box">
       <div class="cart-total-box">
         <p class="p-large text-left text-bold-weight">Price Details</p>
         <hr />
         <div class="side-ways-cart">
-          <p class="p-large text-left">Price (2 items)</p>
-          <p class="p-large text-left">Rs. 1198</p>
+          <p class="p-large text-left">Price ({item.quantity} items)</p>
```

Comment:
align-text property is inherit property so no need to give for every child instead you can give to parent directly